crates/gen-guest-teavm-java/tests/codegen.rs

```diff
@@ -4,18 +4,21 @@ use std::path::Path;
 use std::process::Command;
 
 macro_rules! codegen_test {
-    ($name:ident $test:tt) => {
+    // TODO: should get this test working
+    (use_across_interfaces $($x:tt)*) => {};
```

Comment:
Oh @dicej I also meant to point this out to you, as your assistance would be appreciated in getting this test working. This has to do with importing things from different paths which I'm less certain how it works in Java. Again happy to have you walk me through it over Zoom or similar or can provide more details myself as well.

Reply:
Sure, I'll take a look -- might need to wait until Monday, though.